roboteam_ai/src/utilities/GameStateManager.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ * Refstate commands are given from the robocup as integers
```

Comment:
They're an enum in ProtoBuf iirc. We can also change the way we send them over ROS.